test/src/100-publishstatistics/main

```diff
@@ -0,0 +1,243 @@
+# This file contains $NR_OF_TESTS tests for checking the publish statistics feature
+cvmfs_test_name="Publish statistics"
+
+ERROR_FILES_ADDED=101
+ERROR_FILES_REMOVED=102
+ERROR_FILES_CHANGED=103
+ERROR_DIR_ADDED=104
+ERROR_DIR_REMOVED=105
+ERROR_DIR_CHANGED=106
+ERROR_BYTES_ADDED=107
+ERROR_BYTES_REMOVED=108
+
+NR_OF_TESTS=4
+
+TAR_FILE=tarfile.tar
+TAR_DIR=newTarDir
+
+test1() {
+  local working_dir=$1
+
+  pushdir $working_dir
+
+  echo "meaningless file content" > file1    # 1 file added ; 25 bytes added
+  echo "more clever file content" > file2    # 1 file added ; 25 bytes added
+  echo "" > file3                            # 1 file added ; 1 byte added ['\n']
+  mkdir foo                                  # 1 directory added
+  mkdir bar                                  # 1 directory added
+  touch ./foo/file4                          # 1 file added ; 0 byte added
+  echo "123 - testing" > ./foo/file5         # 1 file added ; 14 byte added
+  ln -s file1 symlinkTofile1                 # 1 file added ; 25 bytes added
+  ln file2 hardlinkTofile2                   # 1 file added ; 25 bytes added
+
+  popdir
+}
+
+test2() {
+  local working_dir=$1
+
+  pushdir $working_dir
+
+  echo "additional meaningless content" >> file1   # 1 file changed ; 31 bytes added
+  echo "new file content" > file2                  # 1 file changed ; 8 bytes removed
+  touch bar                                        # 1 directory changed
+  rm -rf foo               # 1 directory removed ; 2 files removed ; 14 bytes removed
+
+  popdir
+}
+
+#ingest command test
+test3() {
+  local working_dir=$1
+
+  pushdir $working_dir
+  rm symlinkTofile1
+  rm hardlinkTofile2
+  echo "Making tar file..."
+  tar -cvf $TAR_FILE * || return $?
+  echo "Tar file created"
+
+  popdir
+}
+
+cvmfs_run_test() {
+  logfile=$1
+  local repo_dir=/cvmfs/$CVMFS_TEST_REPO
+  local scratch_dir=$(pwd)
+  mkdir reference_dir
+  local reference_dir=$scratch_dir/reference_dir
+
+  echo "Create a fresh repository named $CVMFS_TEST_REPO with user $CVMFS_TEST_USER"
```

Comment:
Prefixing the explicit echo statements with `*** ` makes it easier to navigate through the logs (we haven't done this in all the tests yet)